rql/src/app.rs

```diff
@@ -430,4 +419,15 @@ impl App {
             _ => false,
         }
     }
+
+    fn intersperse_keys<'a, 'b>(
+        keys: &'a [&'b str],
```

Comment:
neat, i'm not used to seeing lifetimes like this

Reply:
I forgot how I worked myself into this hole. I think I wanted to allow the passed strs to have separate lifetimes :thinking:.

Anyway, this can be all one lifetime and looks better if it is.